public/app/features/plugins/admin/routes.tsx

```diff
@@ -10,18 +10,21 @@ const DEFAULT_ROUTES = [
   {
     path: '/plugins',
     navId: 'plugins',
+    roles: () => ['Admin'],
```

Comment:
I think the legacy access-control is slightly more complex than this, the plugins NavLink is displayed with:
`rc.OrgRole == org.RoleAdmin || cfg.PluginAdminEnabled && rc.IsGrafanaAdmin`

Someone with `Grafana Admin` rights not being the current `Org Admin` cannot access the NavLink any more (I agree this is a strange use case :sweat_smile:)
![edge case](https://user-images.githubusercontent.com/5232696/196625733-b95bd70b-204e-4df7-a321-e80b0836a5d9.gif)

Reply:
@gamab Thanks for pointing this out. This [commit](https://github.com/grafana/grafana/pull/57101/commits/d0b540617089c5b195b01d2ee71991e5e5b3e3ae) should fix the issue.

Reply:
Thanks!